### ISSUE TITLE
Deprecated middleware warning when running gulp

### ISSUE THREAD

**Issue by Bouncey** [FreeCodeCamp/FreeCodeCamp#11188]
Is there a replacement package for this?

``` sh
strong-remoting deprecated The built-in CORS middleware provided by REST adapter was deprecated. See https://docs.strongloop.com/display/public/LB/Security+considerations for more details. node_modules/strong-remoting/lib/remote-objects.js:97:25
```

**Comment by kp2356** [comment]
http://loopback.io/doc/en/lb2/Security-considerations.html#deprecation-of-built-in-cors-middleware

Check for more information

**Comment by marhyorh** [comment]
Maybe we'd have to add [CORS](https://www.npmjs.com/package/cors)

**Comment by zalos** [comment]
Any word on this? Im getting issues deploying to Heroku. Im looking into this as well.

**Comment by systimotic** [comment]
I'm no longer getting this running the latest staging. I believe this was fixed-but-broken, then fixed again in #11415
@Bouncey Can this be closed?
